viewer_server/client_db_access_server.py

```diff
@@ -312,7 +319,7 @@ def process_request(self, request, client_address):
 
 
 def start_server(package_data, port, db_conn_string, suppress_handler,
-                 not_host_only, db_version_info):
+                 not_host_only, context):
```

Comment:
No, the context carries the `codechecker_workspace` information into the server.